scripts/player.gd

```diff
@@ -0,0 +1,22 @@
+extends KinematicBody2D
+
+# Declare member variables here. Examples:
+var speed = 200
+var velocity = Vector2()
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	# Player movement
+	velocity = Vector2()
+
+	if Input.is_action_pressed("ui_right"):
```

Comment:
I think we should make new actions for move_right and move_left instead of overloading ui_direction